haproxy-ingress/templates/_podtemplate.yaml

```diff
@@ -108,6 +108,10 @@ spec:
         - name: haproxy-template
           mountPath: /etc/templates/haproxy
 {{- end }}
+{{- if .Values.controller.customFiles }}
+        - name: haproxy-custom-files
+          mountPath: /etc/haproxy-custom
+{{- end }}
```

Comment:
This needs to be `{{- if and .Values.controller.customFiles (not .Values.controller.haproxy.enabled) }}` so you'll only mount the files if haproxy is embedded. Add this same config into the haproxy container.

Reply:
Better `/etc/haproxy-custom-files` instead.

Reply:
done!

Reply:
Missing the mount point in the haproxy container, note that it's enough to place inside `{{- if .Values.controller.customFiles }}` only. The same applies to the `volumes` config. In the end you should have one mount point, either in the controller (if embedded) or haproxy (if external) container, and the volumes declaration that should always exist.

Reply:
added to haproxy container